app/models/user.rb

```diff
@@ -33,4 +33,16 @@ class User < ApplicationRecord
   def over_ta?
     %w[ta admin super].include? role
   end
+
+  def ensure_authentication_token
+   loop do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Layout/IndentationWidth: Use 2 (not 1) spaces for indentation.